src/main/java/com/mrfloris/boosters/pluginEvents.java

```diff
@@ -20,7 +22,19 @@
     public static String isInactive;
     public static String isActive;
     public static Boolean isDebug;
+    Pattern pattern = Pattern.compile("#[a-fA-F0-9]{6}");
     public String color(String msg) {
+        if (!Bukkit.getVersion().contains("1.8") && !Bukkit.getVersion().contains("1.9")
+                && !Bukkit.getVersion().contains("1.10") && !Bukkit.getVersion().contains("1.11")
+                && !Bukkit.getVersion().contains("1.12") && !Bukkit.getVersion().contains("1.13")
+                && !Bukkit.getVersion().contains("1.14") && !Bukkit.getVersion().contains("1.15")) {
+            Matcher matcher = pattern.matcher(msg);
+            while (matcher.find()) {
+                String color = msg.substring(matcher.start(), matcher.end());
+                msg = msg.replace(color, ChatColor.of(color) + "");
+                matcher = pattern.matcher(msg);
+            }
+        }
```

Comment:
I belive this if statement can now be removed then, 
as below?
```suggestion
        Matcher matcher = pattern.matcher(msg);
        while (matcher.find()) {
            String color = msg.substring(matcher.start(), matcher.end());
            msg = msg.replace(color, ChatColor.of(color) + "");
            matcher = pattern.matcher(msg);
        }
```

Reply:
Code's too confusing for my noob brain. I'd have to test if it works first, if so, ill keep it and ill just update the readme that this for 1.16.5 specifically and older versions aren't supported once i release the next jar